plugins/net_plugin/net_plugin.cpp

```diff
@@ -201,6 +240,9 @@ namespace eosio {
       bool have_block(const block_id_type& blkid) const;
       void rm_block(const block_id_type& blkid);
 
+      void add_unlinkable_block( signed_block_ptr b, const block_id_type& id );
+      unlinkable_block_state get_possible_linkable_block(const block_id_type& blkid);
```

Comment:
Should `get_possible_linkable_block` be named something like `get_and_remove_possible_linkable_block` as it removes the block from the cache?

Reply:
Renamed to `pop_possible_linkable_block`.